Catappult/appcoins-wallet-android
# [feature/APPC-3339] Blacklisted wallet backup

**What does this PR do?**

   Adds the backup as an option in the blacklisted versions if there is a wallet available.
   If there are no wallets, the backup option will not appear.

**Database changed?**

   No

**Where should the reviewer start?**

- UpdateRequiredFragment.kt
- IabUpdateRequiredFragment.kt
- 4 xml files

**How should this be manually tested?**

- Blacklist the current version (maybe ask the backend to blacklist in dev, in prod, we can release 2 versions to blacklist one)
- Check the 4 layouts (in the app and in the payment flow, and in landscape for both)
- Check the backup flow and its navigation

**What are the relevant tickets?**

  https://aptoide.atlassian.net/browse/APPC-3339

**Code Review Checklist**
